SharedProjects/Babylon2GLTF/GLTFExporter.Material.cs

```diff
@@ -290,7 +293,7 @@ private void ExportMaterial(BabylonMaterial babylonMaterial, GLTF gltf)
                                 opacityBitmap = TextureUtilities.LoadTexture(babylonStandardMaterial.opacityTexture.originalPath, logger);
                             }
 
-                            if (diffuseBitmap != null || specularBitmap != null || opacityBitmap != null)
+                            if ((diffuseBitmap != null && (specularBitmap != null || opacityBitmap != null)) || specularBitmap != null || opacityBitmap != null)
```

Comment:
This change was made to avoid having to generate a new texture if there are no other diffuse textures to pack into it.

The cases are:

There is a diffuse bitmap and a specular or opacity bitmap to be packed into the resultant base color bitmap.

There is no diffuse bitmap, but there is a specular or opacity bitmap to be packed into the base color bitmap.